### DISCUSSION TITLE
stt_hi_conformer_ctc_large model is not available under NeMo

### DISCUSSION THREAD

**Discussion by Munikumar09** [NVIDIA-NeMo/NeMo#8031] [Q&A] [answered]
I am developing Hindi automatic speech recognition (ASR) systems and I found the NVIDIA [Riva](https://docs.nvidia.com/deeplearning/riva/user-guide/docs/tutorials/New-language-adaptation/Hindi/README.html) Hindi ASR models on their website. They claim to have trained two conformer models: [Conformer medium](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/nemo/models/stt_hi_conformer_ctc_medium) and [Conformer large](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/nemo/models/stt_hi_conformer_ctc_large) models. However, when I use the NeMo toolkit, I can only access the Conformer medium model. The Conformer large model is not available. 
This is the Python code I used:
```
import nemo.collections.asr as nemo_asr
asr_model = nemo_asr.models.EncDecCTCModelBPE.from_pretrained(model_name="stt_hi_conformer_ctc_large")
```
This is the error message I received:
```
 FileNotFoundError: Model stt_hi_conformer_ctc_large was not found. Check cls.list_available_models() for the list of all available models.
```
Can anyone explain why this is happening and how to fix it?

**Comment by titu1994** [comment, marked as answer]
Seems they were not added to NeMo repo. That isn't a big issue, you can download it from here and use it with restore_from() instead https://catalog.ngc.nvidia.com/orgs/nvidia/teams/nemo/models/stt_hi_conformer_ctc_large
